### ISSUE TITLE
data = dataiter.next() is giving an attribute error that next attribute isn't found

### ISSUE THREAD

**Issue by satichandrala** [python-engineer/pytorchTutorial#22]
- I'm using a Windows 10 Pro OS and in the data loader, dataiter.next() isn't working. 
`Exception has occurred: AttributeError
'DataLoader' object has no attribute 'next'
    File "C:\Users\xxx\Desktop\xxxx\xxxxx\code\pytorchbasics\datasetanddataloaderclass.py", line 40, in <module>
    datatiter = dataloader.next() `

**Comment by satichandrala** [comment]
However, If anyone's facing some run time errors, when starting to print the features and labels after the data loader step, add if __name__ == '__main__': , and after dataiter,  use  data = next(datatiter) if dataiter.next() isn't working. Not sure if this is the same for all, but I use VS Code on a Windows OS and I was getting a run time error without a name =  main and an attribute error for next. 
Hope it's helpful, you can also refer https://stackoverflow.com/questions/18204782/runtimeerror-on-windows-trying-python-multiprocessing, for understanding the runtime error  and https://stackoverflow.com/questions/74289077/attributeerror-multiprocessingdataloaderiter-object-has-no-attribute-next, to understand the error with dataiter.next()

**Comment by satichandrala** [comment]
after dataiter, use data = next(datatiter) if dataiter.next() isn't working. 
Reference: 
 https://stackoverflow.com/questions/74289077/attributeerror-multiprocessingdataloaderiter-object-has-no-attribute-next, to understand the error with dataiter.next()
